client/api/src/main/java/org/opennms/velocloud/client/api/model/EnterpriseEvent.java

```diff
@@ -0,0 +1,147 @@
+package org.opennms.velocloud.client.api.model;
+
+import java.time.OffsetDateTime;
+import java.util.StringJoiner;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class EnterpriseEvent {
+    private String detail;
+    private String category;
+    private String event;
+    private Integer id;
+    private OffsetDateTime eventTime;
+    private String edgeName;
+    private String enterpriseUsername;
+    private String message;
+    private String severity;
+
+    private EnterpriseEvent(final Builder builder) {
```

Comment:
Is there any of this fields required to be not null?

Reply:
done